### PR TITLE
use publish version instead of git master

### LINKED ISSUE #634: Using master and not releases of ethereumjs libraries is dangerous
#432 has changed all those imports:

```diff
 -    "ethereumjs-block": "^1.2.2",
 -    "ethereumjs-tx": "^1.1.1",
 -    "ethereumjs-util": "^4.4.0",
 -    "ethereumjs-vm": "^2.0.1",
 +    "ethereumjs-block": "https://github.com/ethereumjs/ethereumjs-block",
 +    "ethereumjs-tx": "https://github.com/ethereumjs/ethereumjs-tx",
 +    "ethereumjs-util": "https://github.com/ethereumjs/ethereumjs-util",
 +    "ethereumjs-vm": "https://github.com/ethereumjs/ethereumjs-vm",
```

Any bug making it to master of these libraries can cause an outage in Remix.

### PR DESCRIPTION
**Pull request by yann300**
Fixes #634.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,11 @@
     "csjs-inject": "^1.0.1",
     "csslint": "^1.0.2",
     "ethereum-remix": "https://github.com/ethereum/remix",
-    "ethereumjs-abi": "https://github.com/ethereumjs/ethereumjs-abi",
-    "ethereumjs-block": "https://github.com/ethereumjs/ethereumjs-block",
-    "ethereumjs-tx": "https://github.com/ethereumjs/ethereumjs-tx",
-    "ethereumjs-util": "https://github.com/ethereumjs/ethereumjs-util",
-    "ethereumjs-vm": "https://github.com/ethereumjs/ethereumjs-vm",
+    "ethereumjs-abi": "^0.6.4",
```
Comment (axic) [inline package.json]: The ABI was not released yet so you might want to keep this on master.

File: package.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
     "csslint": "^1.0.2",
     "ethereum-remix": "https://github.com/ethereum/remix",
     "ethereumjs-abi": "https://github.com/ethereumjs/ethereumjs-abi",
-    "ethereumjs-block": "https://github.com/ethereumjs/ethereumjs-block",
-    "ethereumjs-tx": "https://github.com/ethereumjs/ethereumjs-tx",
-    "ethereumjs-util": "https://github.com/ethereumjs/ethereumjs-util",
-    "ethereumjs-vm": "https://github.com/ethereumjs/ethereumjs-vm",
+    "ethereumjs-block": "^1.6.0",
+    "ethereumjs-tx": "^1.3.3",
+    "ethereumjs-util": "^5.1.2",
```
Comment (axic) [inline package.json]: Oh, I think this is the broken version according to the PRs, but not sure.

Comment (yann300) [inline package.json]: I checked https://github.com/ethereum/browser-solidity/issues/633 and https://github.com/ethereum/browser-solidity/issues/628 
both are fixed (even using util 5.1.2 ...)

Comment (axic) [inline package.json]: Great! :)

**Comment by GoldenDave** [comment]
Thanks.

**Pull request merged**